package.json

```diff
@@ -29,31 +29,43 @@
     "Google Cloud Key Management Service (KMS) API"
   ],
   "scripts": {
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "test": "c8 mocha",
-    "lint": "eslint '**/*.js'",
+    "test": "c8 mocha build/test",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "system-test": "mocha build/system-test",
+    "lint": "gts fix && cd samples/ && npm install && cd .. && eslint --fix samples/*.js",
```

Comment:
`lint` and `fix` are a bit mixed up. 
- We shouldn't do the `cd samples/ && npm install && cd ..` bit here for lint.  There is a `prelint` command floating around we've started to use everywhere
- This seems to do `eslint --fix` in the `lint`.  It should just be `eslint samples/*.js`
- `fix` needs the `&& eslint --fix` bit.

Reply:
great suggestion, modifying it, that will give cleaner script i can see, thanks.